raml-parser-2/src/main/java/org/raml/v2/internal/impl/commons/phase/IncludeResolver.java

```diff
@@ -130,6 +133,9 @@ private void setTempPositionWithResourceUri(Node result)
         startPosition.setIncludedResourceUri(includedResourceUri);
         endPosition.setIncludedResourceUri(includedResourceUri);
 
+        startPosition.setIncludedResourcePath(includedResourcePath);
+        endPosition.setIncludedResourcePath(includedResourcePath);
+
```

Comment:
What's the difference between `includedResourceUri` and `includedResourcePath`? Why do we need both of them?